pending/aleksei_sentry_added

```diff
@@ -0,0 +1 @@
+[Added] [#3048](https://github.com/cosmos/lunie/issues/3048) Sentry integration was added @iambeone
```

Comment:
This is the wrong path for the changelog. Did you run `yarn changelog`?

Reply:
Yes

Reply:
Fixed it for you. As you can see here https://github.com/luniehq/lunie/blob/develop/package.json#L11 the entry should be created in ./changes. Maybe there is an issue on Windows? 🤷‍♂

Reply:
But it works for extension Oo